instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/context/ContextPropagationDebug.java

```diff
@@ -22,9 +23,13 @@
       ContextKey.named("thread-propagation-locations");
 
   private static final boolean THREAD_PROPAGATION_DEBUGGER =
-      Boolean.getBoolean("otel.javaagent.experimental.thread-propagation-debugger.enabled");
+      Config.get()
+          .getBooleanProperty(
+              "otel.javaagent.debug.thread-propagation.enabled",
```

Comment:
Don't we need the experimental still? If I'm not mistaken, actually this flag is supposed to go away at some point since the SDK includes a similar flag now.

Reply:
makes sense, reverted name, but kept the new default